org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/ToolbarCustomButton.java

```diff
@@ -41,6 +42,16 @@ public ToolbarCustomButton(MToolBarButton mToolbarButton, Toolbarbutton btn, Str
 		toolbarButton.addEventListener(Events.ON_CLICK, this);
 	}
 	
+	public ToolbarCustomButton(MToolBarButton mToolbarButton, Toolbarbutton btn, String actionId, int windowNo, int tabNo) {
```

Comment:
to reduce duplication of code, the old constructor should call the new constructor with tabNo=-1

Reply:
Fixed